.gitignore

```diff
@@ -54,5 +54,13 @@ nbproject
 /tests/coverage*
 /tests/clover.xml
 /tests/js/node_modules
+/tests/output/
 
 build
+
+.php_cs.cache
+
+# Composer
+composer.phar
+vendor/
+vendor-bin/**/composer.lock
```

Comment:
Also add:
```
vendor-bin/**/vendor
```
like I just did in https://github.com/owncloud/files_antivirus/pull/273/files#r235873391

Some day we will have all the pieces of the puzzle in one place.